src/UI/Actions.hs

```diff
@@ -731,7 +732,7 @@ manageMailTags
     -> (Int, NotmuchMail)
     -> m (AppState -> AppState)
 manageMailTags s tagop m =
-  either setError updateMails <$> applyTagOps tagop (Vector.singleton m) s
+  either setError updateMails <$> applyTagOps tagop [m] s
```

Comment:
@romanofski random comment about this: `(Identity m)` would work fine and, unlike `[]` or `Vector`
has no runtime overhead.  But I was not bothered with it :)